### DISCUSSION TITLE
CLI install docs doesn't mention aliasing

### DISCUSSION THREAD

**Discussion by kristian-nst** [oven-sh/bun#14938] [General]
There's this page: https://bun.sh/guides/install/npm-alias
![image](https://github.com/user-attachments/assets/f8d1558e-35a0-4689-8c05-e02ad7b2fda0)

It's linking you to the full docs (https://bun.sh/docs/cli/install), but the full docs doesn't have this aliasing section anywhere 😂

**Comment by RiskyMH** [comment]
Fixed in #15754, let me know if you wanted more (I think it's pretty basic)
